packages/spor-react/src/input/RadioCard.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  Radio as ChakraRadio,
+  RadioProps as ChakraRadioProps,
+  forwardRef,
+  useRadio,
+  useRadioGroupContext,
+} from "@chakra-ui/react";
+import React, { FormEvent } from "react";
+import { Card } from "../card";
+import { Text } from "../typography";
+import { TextProps as ChakraTextProps } from "@chakra-ui/layout/dist/text";
+import { Stack } from "../layout";
+import { getBoxShadowString } from "../theme/utils/box-shadow-utils";
+
+export type RadioCardProps = Exclude<
+  ChakraRadioProps,
+  "colorScheme" | "size" | "variant"
+>;
+
+/**
+ * Radio buttons that look like cards.
+ *
+ * Specify the label in a `<RadioCardHeader>` and the value as `value`.
+ *
+ * ```tsx
+ * <RadioCard value="#f00"><RadioCardHeader>Red</RadioCardHeader></RadioCard>
+ * ```
+ *
+ * You typically want to place RadioCard components in a group with several other RadioCard components. You can do that with the `RadioGroup` component.
+ *
+ * ```tsx
+ * <RadioGroup name="ticket">
+ *   <RadioCard value="economy">
+ *     <RadioCardHeader>Economy</RadioCardHeader>
+ *   </RadioCard>
+ *   <RadioCard value="business">
+ *     <RadioCardHeader>Business</RadioCardHeader>
+ *   </RadioCard>
+ *   <RadioCard value="first-class">
+ *     <RadioCardHeader>First Class</RadioCardHeader>
+ *   </RadioCard>
+ * </RadioGroup>
+ */
+export const RadioCard = forwardRef<RadioCardProps, "input">((props, ref) => {
+  const group = useRadioGroupContext();
+
+  const isChecked =
+    group?.value !== null &&
+    props.value !== null &&
+    group.value === props.value;
+
+  const { getInputProps } = useRadio({
+    ...props,
+    isChecked,
+    name: group?.name,
+    onChange: group?.onChange,
+  });
+
+  const inputProps = getInputProps();
+
+  return (
+    <Card
+      as="label"
```

Comment:
Er det noen nedsider med å gjøre hele kortet til en label? Det kan hende vi burde tillate folk å spesifisere `aria-label` i alle fall, så man kan gi en litt mer konsis label til valget enn det man ellers ville sett. Man burde kanskje få lov å sende inn resten av propsa til card også – så komponenten blir så fleksibel som mulig

Reply:
Den eneste nedsiden er det med at alt blir lest opp av en skjermleser (som enten er en nedside eller en fordel, avhengig av hvordan du ser det). Men det gjør også at man ikke kan ha klikkbare elementer inni kortet. Så det støtter ikke denne enkle versjonen av radio cards

Reply:
Hm okei. Mulig man kan lage en litt mer sammensatt versjon med flere subkomponenter, som lar oss sette sammen mer kompliserte versjoner ved behov. Jeg skal tenke litt på potensielle versjoner.

Reply:
Hva er et usecase for et klikkbart element inni et radiokort, egentlig? 🤔

Reply:
Det er jo ikke helt bra å ha sånn kort med klikkbare elementer, men kjøpsløpet bruker det en del med lenker og knapper inni radio cards.